deploy.sh

```diff
@@ -57,22 +66,22 @@ function retry_terraform {
     cmd=$2
     for i in $(seq 1 "$tries"); do
         fatal_errors=()
-        LOGFILE=$LOGFILE_$i
+        LOGFILE=${LOGFILE}_$i
         echo "Attempt: $i/$tries"
         echo "========================" >>$LOGFILE
         echo "Attempt: $i/$tries" >>$LOGFILE
         echo "$cmd" >>$LOGFILE
         echo "========================" >>$LOGFILE
         $cmd >>$LOGFILE 2>&1 &
         tpid=$!
-
-        while $(ps | grep "$tpid"); do
+        loop_itr=$(ps | grep "$tpid")
```

Comment:
Was there any issue with existing condition?

Reply:
existing while loop condition was giving `command not found error` on CentoOS